skrypty/inventory/special_inventory.lua

```diff
@@ -1,3 +1,8 @@
+scripts.inv.get_magics_to_put_down_exempts = {
```

Comment:
+ gadacz

Reply:
Właśnie nad gadaczem się zastnawiałem. Dodam go, w razie czego można sobie to samodzielnie nadpisać w pliku imie.txt